packages/webapp/src/pages/index.tsx

```diff
@@ -40,59 +62,50 @@ export const Index = () => {
+                                protects and enhances the independence and power of
+                                those who join. When you join the Eden community, you
+                                gain access to a group of people working together to
+                                empower you and your family to make a bigger impact in
+                                the world.
+                            </Text>
                             <Text>
-                                The community has{" "}
-                                <strong>
-                                    {memberStats.active_members} active member
-                                    {memberStats.active_members !== 1 && "s"}
-                                </strong>{" "}
-                                and{" "}
-                                <strong>
-                                    {memberStats.pending_members} pending
-                                    invitation
-                                    {memberStats.pending_members !== 1 && "s"}
-                                </strong>
+                                To learn more about Eden and how you can get involved,
+                                visit{" "}
+                                <Link
+                                    href="http://eden.eoscommunity.org"
+                                    target="_blank"
+                                    isExternal
+                                >
+                                    eden.eoscommunity.org
+                                </Link>
                                 .
                             </Text>
-                        )}
-                    </div>
-                    <div className="col-span-2 md:col-span-1 space-y-4">
-                        <Text>
-                            EdenOS is a revolutionary new democratic process
-                            that protects and enhances the independence and
-                            power of those who join. When you join the Eden
-                            community, you gain access to a group of people
-                            working together to empower you and your family to
-                            make a bigger impact in the world.
-                        </Text>
-                        <Text>
-                            To learn more about Eden and how you can get
-                            involved, visit{" "}
-                            <Link
-                                href="http://eden.eoscommunity.org"
-                                target="_blank"
-                                isExternal
-                            >
-                                eden.eoscommunity.org
-                            </Link>
-                            .
-                        </Text>
-                    </div>
+                        </div>
+                        </div>
```

Comment:
The spacing/indentation's still messed up on this file. Is Prettier autoformatting on save for you?